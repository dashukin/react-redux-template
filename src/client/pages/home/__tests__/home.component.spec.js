import React from 'react';
import { shallow } from 'enzyme';
import HomePage from '../home.component';

describe('Home page', () => {
  const mockTranslate = jest.fn(value => value);
  it('should render Home page', () => {
    const wrapper = shallow(<HomePage translate={mockTranslate} />);

    expect(wrapper).toMatchSnapshot();
  });
});
