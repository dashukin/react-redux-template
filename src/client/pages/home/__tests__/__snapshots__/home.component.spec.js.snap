// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home page should render Home page 1`] = `
<div>
  app.homepage.title
  <p>
    Home page content
  </p>
</div>
`;
